researcher-reports/src/components/query-item.tsx

```diff
@@ -1,5 +1,7 @@
 import React, { useState, useEffect } from "react";
 import * as AWS from "aws-sdk";
+// eslint-disable-next-line @typescript-eslint/no-require-imports
+const AmazonS3URI = require("amazon-s3-uri");
```

Comment:
Did you try `import AmazonS3URI from "amazon-s3-uri";`

Reply:
Ah, I thought I tried that, but I must not have.  I must have been too hung up on the syntax that they presented in their example.  Fixed.